tests/CMakeLists.txt

```diff
@@ -146,24 +152,26 @@ FOREACH(_test ${_tests})
     SET(_use_test OFF)
   ENDIF()
 
-  SHOULD_ENABLE_TEST(${CMAKE_CURRENT_SOURCE_DIR}/${_test}.prm)
+  SHOULD_ENABLE_TEST(${CMAKE_CURRENT_SOURCE_DIR}/${testname}.prm)
 
   IF(_use_test)
 
+    MESSAGE(STATUS "** processing test ${testname}:")
+
     # Create main target for this test. We let it depend on the screen output
     # even if the folder with reference data for this test doesn't contain any
     # files. This is useful when you are constructing a new test.
-    ADD_CUSTOM_TARGET(tests.${_test}
-	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${_test}/screen-output)
+    ADD_CUSTOM_TARGET(tests.${testname}
+	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/output-${testname}/screen-output)
 
     MATH(EXPR _n_tests "${_n_tests} + 1")
 
-    IF(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${_test}.cc)
-      ADD_LIBRARY(${_test} SHARED EXCLUDE_FROM_ALL ${_test}.cc)
-      ASPECT_SETUP_PLUGIN(${_test})
+    IF(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${testname}.cc)
+      ADD_LIBRARY(${testname} SHARED EXCLUDE_FROM_ALL ${testname}.cc)
+      ASPECT_SETUP_PLUGIN(${testname})
 
-      SET(_testlib 'set Additional shared libraries = ./lib${_test}.so')
-      SET(_testdepends ${_test})
+      SET(_testlib 'set Additional shared libraries = ./lib${testname}.so')
+      SET(_testdepends ${testname})
```

Comment:
what I'm getting at is that it's weird that the *target* for the shared lib is called the same as the test.